### DISCUSSION TITLE
VRCOSC: Modular OSC application creator

### DISCUSSION THREAD

**Discussion by VolcanicArts** [vrchat-community/osc#128] [Show and tell]
VRCOSC is a modular OSC application creator to allow for people to quickly create and publish OSC apps to a centralised source. It provides a GUI for ease of use, debugging tools, and comes with prefabs complete with guides on how to use the prefabs on avatars. The wiki page contains extensive documentation on how to create a module and the application is constantly being updated.

The current list of modules can be found [here](https://github.com/VolcanicArts/VRCOSC#modules). New modules are constantly being added as well as prefabs to pair them.

GitHub Repo: https://github.com/VolcanicArts/VRCOSC

**Comment by momo-the-monster** [comment]
@VolcanicArts - thanks for sharing! Your tool has been added to [the official listing](https://github.com/vrchat-community/osc/wiki/Resources#utilities).
